### PR TITLE
added md5sums

### PR DESCRIPTION
**Pull request by Gerbenvandervries**

### CODE REVIEW

File: easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+easyblock = "Tarball"
+
+name = 'SNP2HLA'
+version = 'v1.0.3'
+
+homepage = 'http://software.broadinstitute.org/mpg/snp2hla/'
+description = ''
```
Comment (pneerincx) [inline easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb]: Please add at least a one line description.

File: easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://software.broadinstitute.org/mpg/snp2hla/data/']
+sources = ['SNP2HLA_package_v1.0.3.tar.gz']
```
Comment (pneerincx) [inline easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb]: Please use name and version variables to compile the sources file name(s).

File: easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+checksums = ['b8ca677860e37f4437f54bd3bd475ade']
+
+#postinstallcmds = ['cd %(installdir)s/source ',
+#	'g++ -O5 -o cranefoot main.cc *.cpp -lm'
+#	]
+
+sanity_check_paths = {
+    'files': ["README.txt"],
```
Comment (pneerincx) [inline easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb]: Please test at least for the installed binary. Making sure the README is there is nice, but in this case the installation may silently fail to install the actual software.

File: easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+homepage = 'http://software.broadinstitute.org/mpg/snp2hla/'
+description = ''
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://software.broadinstitute.org/mpg/snp2hla/data/']
+sources = ['SNP2HLA_package_v1.0.3.tar.gz']
+checksums = ['b8ca677860e37f4437f54bd3bd475ade']
+
+#postinstallcmds = ['cd %(installdir)s/source ',
```
Comment (pneerincx) [inline easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb]: If this commented stuff does nothing useful, please remove it.

File: easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb (hunk 1 above)
Comment (Gerbenvandervries) [inline easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb]: done

File: easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb (hunk 2 above)
Comment (Gerbenvandervries) [inline easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb]: done

File: easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb (hunk 3 above)
Comment (Gerbenvandervries) [inline easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb]: added extra folder were tool is installed.

File: easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb (hunk 4 above)
Comment (Gerbenvandervries) [inline easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb]: done

**Pull request merged**